### ISSUE TITLE
Builtin Eye-Tracking Support

### ISSUE THREAD

**Issue by kazu0617** [hazre/VRCFTReceiver#2]
This mod now have to configure avatar's, but resonite already support eye tracking without any configure avatar's if support eyetracker now.

So if can, I want to use this mod with resonite builtin Eye-Tracking solution.

**Comment by hazre** [comment]
You're right, I should just use the eye manager for eye tracking stuff, it wouldn't change much.

I will look into it.

**Comment by sveken** [comment]
+1. 
I have my avatar mostly setup using this method now due to the Crystal eye tracker not being supported in reso. 
Though i need to dig in further to make the pivot swing more. 

Being able to use the inbuilt eye tracker that has the right values all ready to go would make this and future avatars much more straight forward.

**Comment by hazre** [comment]
I've had a local build with built-in eye tracking support for a while but I ran into a issue where the eyes would flicker like crazy when gazing at edge of the eyes. I try to see if I can fix that, so I can release it.

**Comment by sveken** [comment]
> I've had a local build with built-in eye tracking support for a while but I ran into a issue where the eyes would flicker like crazy when gazing at edge of the eyes. I try to see if I can fix that, so I can release it.

Interesting, i thought that was just my headset but i get this with the current set up. Looking far right or left and sometimes the eye will flicker between the side and center. 

Is that what you are seeing using the eye manager?

**Comment by hazre** [comment]
Just an update. I've picked up this again. I'm very close to a release candidate. If you would like test pre-release builds, I put them in this [thread](https://discord.com/channels/901126079857692714/1267910572880429068) in FrooxEngine modding discord.
